examples/apps/inside_out_demo/workload/inside_out_eval_logic.cpp

```diff
@@ -0,0 +1,111 @@
+/* Copyright 2019 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include <string>
+#include <string.h>
+#include <stdint.h>
+#include <stdarg.h>
+#include <vector>
+
+#include "inside_out_eval_logic.h"
+#include "trusted_iohandler.h"
+#include "enclave_utils.h"
+
+size_t GetMaxFileSize()
+{
+    return 1024;
```

Comment:
Where does `1024` come from? Is it a dummy value? Or is it a hard limit.
Add a constant definition.

Reply:
Currently its hard coded. Individual workloads can configure this value.

Reply:
Please add a comment. Also add how this can be configured.